app/src/androidTest/java/com/yacgroup/yacguide/database/Common.kt

```diff
@@ -101,9 +101,9 @@ class TestDB {
         )
 
         val ASCENDS = listOf(
-            Ascend(1, ROUTES[0].id, AscendStyle.eONSIGHT.id, 0, 0, 0, ArrayList(), ""),
+            Ascend(1, ROUTES[0].id, AscendStyle.eONSIGHT.id, 2000, 10, 10, ArrayList(), ""),
             Ascend(2, ROUTES[0].id, AscendStyle.eFOLLOWED.id, 0, 0, 0, ArrayList(), ""),
-            Ascend(3, ROUTES[1].id, AscendStyle.ePROJECT.id, 0, 0, 0, ArrayList(), "")
```

Comment:
Why did you replace this date by a "valid" one?

Reply:
I needed some valid dates to test the "order-by-date" functionality.